dcos/dcos-ui
# DCOS-23834 - Data Layer Reference Implementation

Being this a feature branch, we already reviewed the code in separate. If you still feel there are code improvements I can look into it, but the size of the PR makes it harder to code review the code this second time.

Steps to test:
- Run it and check the Packages Repositories
- Test adding, removing, filtering packages. Remember to save the link to the universe before removing it.
- You could also check the requests being fired on the Network tab to see if it is all good.

NOTE:
We still missing 2 or 3 improvements on this ticket, specially typescript, more graphql and etc. but I believe we achieve the goal to remove the Mixins and the stores already, and I dont want this code to get super old without hitting master.
